src/main/java/racingcar/constants/RacingConstant.java

```diff
@@ -0,0 +1,13 @@
+package racingcar.constants;
+
+public class RacingConstant {
+    public static final String CAR_MOVE_CHARACTER = "-";
+
+    public static final String CAR_NAME_PARSING_DELIMETER = ",";
+
```

Comment:
```suggestion
    public static final String CAR_NAME_PARSING_DELIMITER = ",";

```

delimiter입니다! de-limit-er